tests/Integration/Behaviour/Features/Context/Domain/ProductFeatureContext.php

```diff
@@ -94,11 +95,16 @@ public function addProduct(string $productReference, TableNode $table): void
      */
     public function assertLocalizedProperty(string $productReference, string $fieldName, string $localizedValues)
     {
+        $propertyName = $this->getPropertyByFieldName($fieldName);
         $product = $this->getProductByReference($productReference);
         $expectedLocalizedValues = $this->parseLocalizedArray($localizedValues);
 
         foreach ($expectedLocalizedValues as $langId => $value) {
-            if ($value !== $product->{$fieldName}[$langId]) {
+            if ('empty' === $value) {
```

Comment:
I'm not at ease with this it includes some logic (special wording) in the fixture, and in the (unlikely) case we do want to insert `empty` as a value we can't

Maybe it would work without this just by using `"en-US:"` in the scenario? If not maybe a quick fix on `parseLocalizedArray` would allow it

Reply:
I agree it is strange and dull (it was the easiest thing to do though :D). sure ill check how to make something better